src/ColumnSchema.php

```diff
@@ -105,6 +104,14 @@ public function dbTypecast(mixed $value): mixed
      */
     public function phpTypecast(mixed $value): mixed
     {
+        if (is_string($value) && $rangeParser = $this->getRangeParser()) {
+            return $rangeParser->parse($value);
+        }
+
+        if (is_string($value) && $multiRangeParser = $this->getMultiRangeParser()) {
+            return $multiRangeParser->parse($value);
+        }
```

Comment:
```suggestion
```

Move this to `phpTypecastValue()` method in section `return match ($this->getType()) {...}`